airqo-platform/AirQo-api
# Streaming raw measurements, sites and devices into BigQuery

**_WHAT DOES THIS PR DO?_**

Creates jobs that:

- [x] stream raw AirQo, KCCA and weather measurements into BigQuery
- [x] update BigQuery sites and devices

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**

- [PLAT-1076](https://airqoteam.atlassian.net/browse/PLAT-1076)
- [PLAT-1075](https://airqoteam.atlassian.net/browse/PLAT-1075)

**_HOW DO I TEST OUT THIS PR?_**

- To test the functions, Follow instructions in the [Test Functions section of the README file](https://github.com/airqo-platform/AirQo-api/tree/PLAT-1076-update-big-query-devices-sites-and-airqlouds/src/airflow#run-the-main-function)
- To run the stack, follow instructions in the [Docker section of the README file](https://github.com/airqo-platform/AirQo-api/tree/PLAT-1076-update-big-query-devices-sites-and-airqlouds/src/airflow#running-using-docker). You should see the following DAGs:
     - **AirQo-Historical-Raw-Measurements**
     - **Historical-Raw-Weather-Measurements**
     - **Kcca-Historical-Raw-Measurements**
     - **Update-BigQuery-Sites**
     - **Update-BigQuery-Devices**
